src/Compilers/CSharp/Portable/Symbols/AnonymousTypes/AnonymousTypeManager.cs

```diff
@@ -33,9 +33,9 @@ public NamedTypeSymbol ConstructAnonymousTypeSymbol(AnonymousTypeDescriptor type
             return new AnonymousTypePublicSymbol(this, typeDescr);
         }
 
-        public NamedTypeSymbol ConstructAnonymousDelegateSymbol(AnonymousTypeDescriptor typeDescr)
+        public NamedTypeSymbol ConstructAnonymousDelegateSymbol(ImmutableArray<TypeParameterSymbol> containingTypeParameters, AnonymousTypeDescriptor typeDescr)
```

Comment:
>containingTypeParameters [](http://example.com/codeflow?start=100&length=24)

I am not sure how to interpret this name. Type parameters generally do not contain symbols on their own. Could you elaborate what are these type parameters?